corellium/adapter/src/test/kotlin/flank/corellium/adapter/Example.kt

```diff
@@ -85,8 +86,8 @@ fun main() {
         )
 
         println("* Executing tests")
-        api.executeTest(testPlan).collect { line ->
-            print(line)
+        api.executeTest(testPlan).forEach { flow ->
+            launch { flow.collect { line -> println(line) } }
```

Comment:
is there any reason why you did not merge flow? multiple collects is not so common way to do such operation

Reply:
Each flow from the list is representing output from a different android instance (virtual device) which could be treated as a single test suite with many test case results. Each console is available immediately but the results not, so that is why `List<Flow<String>>`. But if you are asking just about printing lines in this example, it's not production code. Just an example use of the current implementation that will be removed soon before MVP finish.

Reply:
But of course, you are right that technically could be a merge of flows instead of collect calls in the loop.

Reply:
so could you change it to 
`api.executeTest(testPlan).merge().collect { line -> println(line) }`?

Reply:
Do not afraid, I will remove it soon, it's not a production code, just an example for testing purposes.

Reply:
I do not see a point to make any code review for this code if all my propositions (then and now) are dropped, because "it is not production code"

Reply:
Pls, notice that this PR was and still is a draft. Drafts by design are not ready to review.

Besides, that here is PR which is removing that example code https://github.com/Flank/flank/pull/1933.